apps/internal-ui/src/components/Chip/types/index.ts

```diff
@@ -0,0 +1,19 @@
+import { ButtonHTMLAttributes, HTMLAttributes } from 'react';
+
+import { ColorVariants } from '@/variants/color/types';
+
+export interface ChipProps
+  extends Pick<
+    ButtonHTMLAttributes<HTMLButtonElement>,
+    'onClick' | 'disabled' | 'className'
+  > {
+  label: string;
+  ariaLabel?: string;
+  color?: ColorVariants;
+  backgroundColor?: ColorVariants;
```

Comment:
```suggestion
  color?: ColorVariants;
  backgroundColor?: ColorVariants;
```
p2: 현재 디자인에서 한 가지 컬러만 정의되어 있으니 우선 현재 디자인에서 사용 중인 컬러만 정의해두고
추후 필요한 컬러 테마가 생길 시 추가하는 방향이 낫지 않을까요?
Typography와 같이 정의되어 있지 않긴 하지만 여러가지 컬러를 사용하게 될 확률이 상대적으로 낮을 것 같습니다! 
(internal 에서도 현재 Default 컬러 테마만 사용하며 이외 색상은 사용하고 있지 않습니다.)

Reply:
넵, 이거 디자인시스템 상으로는 한 가지 컬러만 되어있는데 유선님과 구두로 Typography와 같이 여러 색상이 올 수 있도록 반영하기로 했습니다.

Reply:
생각해 봤는데 당장에 추가할 필요는 없을 것 같네요.. 그냥 지금 색상으로만 지원하겠습니다.